frontend/src/Pages/Teacher/TeacherDashBoard/TeacherDashboard.jsx

```
@@ -8,72 +8,101 @@ import {
+                            <div className="text-center text-red-500 mt-4">{error}</div>
+                        ) : publications.length > 0 ? (
+                            publications.map((pub) => (
+                                <div key={pub.id} className="bg-gray-800 p-3 rounded mb-2 flex justify-between items-center">
+                                    <div className="mr-4">
+                                        <p className="text-white font-semibold">{pub.title}</p>
+                                        <p className="text-gray-400 text-sm">{pub.journal}</p>
+                                    </div>
+                                    <button
+                                        className="text-blue-400 hover:text-blue-600 whitespace-nowrap"
+                                        onClick={() => navigate(`/publications/${pub.id}`)}
+                                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)